backend/read_data.py

```diff
@@ -10,14 +10,37 @@ def read_dataset():
     return pd.read_csv(os.path.join(os.path.dirname(__file__), "data", "women_bias_data.csv"))
 
 def filter_by_factor(data: pd.DataFrame, filtering_factor: str):
```

Comment:
It seems unnecessarily complicated to pass the filtering factor as a single variable, if we were to implmenet more than one category I believe there should be separate variables for filter_age, filter_race, filter_gender.

For now lets just make only one filter, gender is good since thats what out biased dataset is on.

Reply:
Since as you said, the "other" will cause a bug, and also hard coding ranges for the age seems like a really unintuitive UX

Reply:
i agree i think one variable for gender is a good place to start for now